pangram.json

```diff
@@ -16,6 +16,26 @@
         "expected": false
       },
       {
+        "description": "another missing character 'x'",
+        "input": "the quick brown fish jumps over the lazy dog",
+        "expected": false
+      },
+      {
+        "description": "pangram with underscores",
+        "input": "the_quick_brown_fox_jumps_over_the_lazy_dog",
+        "expected": true
+      },
+      {
+        "description": "pangram with numbers",
+        "input": "the 1 quick brown fox jumps over the 2 lazy dogs",
+        "expected": false
```

Comment:
should this be true, since it has all letters a-z? The presence of extra numbers should not cause the expected result to be false, right?